sr_ethercat_hand_config/demos/demo_lo.py

```diff
@@ -33,6 +33,14 @@
              'lh_THJ1': 0.0, 'lh_THJ2': 0.0, 'lh_THJ3': 0.0, 'lh_THJ4': 0.0, 'lh_THJ5': 0.0,
              'lh_WRJ1': 0.0, 'lh_WRJ2': 0.0}
 
+unpack_hand_1 = {'lh_THJ1': 0.0, 'lh_THJ2': 0.0, 'lh_THJ3': 0.0, 'lh_THJ4': 1.2, 'lh_THJ5': 0.0}
+
+# Open the thumb
+joint_states = unpack_hand_1
```

Comment:
this variable assignment is unnecessary, you can just pass the original var to the method